native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/mysql-native/resource-config.json

```diff
@@ -0,0 +1,20 @@
+{
+  "resources": {
+    "includes": [
+      {
+        "pattern": "\\Qcom/mysql/cj/TlsSettings.properties\\E"
+      },
+      {
+        "pattern": "\\Qorg/quartz/core/quartz-build.properties\\E"
```

Comment:
Can we check removing this quartz resource?

Reply:
Removed in [9f68040](https://github.com/ballerina-platform/module-ballerinax-mysql/pull/653/commits/9f68040d7ddf069c53730192d708bfa74bc74e81)